hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/AbstractFindTargetGreedy.java

```diff
@@ -45,7 +44,7 @@
 public abstract class AbstractFindTargetGreedy implements FindTargetStrategy {
   private Logger logger;
   private ContainerManager containerManager;
-  private PlacementPolicy placementPolicy;
+  private PlacementPolicyValidateProxy placementPolicyValidateProxy;
```

Comment:
Can we create an instance of it inside SCM? I think it can be used by balancer and other components as well.

Reply:
look good, will update